### PR TITLE
update staging guide to include validation

### PR DESCRIPTION
**Pull request by ianthpun**
Add the `is-validated` step into the staging guide. https://github.com/onflow/docs/pull/697

### CODE REVIEW

**Review by gregsantos — APPROVED** [review]

File: docs/cadence_migration_guide/staging-guide.mdx [hunk 1]
```diff
@@ -44,3 +44,13 @@ flow-c1 migrate is-staged HelloWorld --network=testnet
 ```
 
 A response of true indicates that your contract has been approved by the Flow Blockchain Testnet network and is ready for the migration process.
+
+### Validate your contract
+
+To validate your contract, execute the following command:
```
Comment (turbolent) [inline docs/cadence_migration_guide/staging-guide.mdx:50]: The sentence below explains what is actually returned, but the phrasing here makes it seem like the command performs the validation when the command is run. Maybe rephrase:

 ```suggestion
### Check the validation status
 
To check the validation status of your contract update, execute the following command:
```

**Review by turbolent — COMMENTED** [review]
Please also sync this with https://github.com/onflow/cadence-lang.org/tree/main/versioned_docs/version-1.0/cadence_migration_guide.

We should maybe add a linter for that
